packages/caido/PKGBUILD

```
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=caido
+pkgname=('caido-cli' 'caido-desktop')
+pkgver=0.52.0
+pkgrel=1
+pkgdesc='A lightweight web security auditing toolkit.'
```

```suggestion
pkgdesc='Intercepting proxy to replay, inject, scan and fuzz HTTP requests.'
```

We are not forced to copy upstream description, I find something like `Intercepting proxy to replay, inject, scan and fuzz HTTP requests` clearer than a generic `web security auditing toolkit`.